README.md

```diff
@@ -1 +1,37 @@
-# dp-search-builder
\ No newline at end of file
+dp-search-builder
+==================
+
+### Getting started
+
+* Run ```brew install kafka```
+
+### Healthcheck
+
+The endpoint `/healthcheck` checks the connection to the database and returns
```

Comment:
Take a look at the health check 'framework' classes added to go-ns when adding a healthcheck.